src/Context/LayersMachineContext.js

```diff
@@ -7,6 +7,9 @@ class LayersMachineContext {
 
   // A { name, data } object, queued for creation of the data's actor
   lastAddedData = null
+
+  // Draw bounding boxes around image and label (if loaded)
+  labelBBoxEnabled = false
```

Comment:
Should this state be associated with the actual image, not in this image aggregating parent?

Reply:
This is now a map of each layer and its bounding box visibility status. That makes sense though. Maybe each layer's status should live with its `LayerActorContext` instead of at the top level here?

Reply:
Yeah. Or the `ImageActorContext`, i'm not sure.

Reply:
I've moved it to the `LayerActorContext` for now as each layer can simply have a boolean `bbox` key associated with it. Moving it to the `ImageActorContext` would be just as simple, we'd just need an `imageBBox`  and `labelBBox` (or something along those lines) associated with each image actor.

Reply:
Most excellent.

In the future, it will also be useful to show the bounding box for point sets, etc.